docs/contributing.md

```diff
@@ -74,18 +74,18 @@ If the feature you're interested in is a JavaScript API, you can cross-reference
 
 Many browsers within BCD can be derived from other browsers given they share the same engine, for example Opera derives from Chrome, and Firefox Android derives from Firefox. To help cut down time working on copying values between browsers, a mirroring script is provided. You can run `npm run mirror <browser> <feature_or_file> [--source=""] [--modify=""]` to automatically copy values.
 
-The <browser> argument is the destination browser that values will be copied to. The script automatically determines what browser to copy from based upon the destination (see table below), but manual specification is possible through the `--source=""` argument.
-
-| Destination      | Default Source    |
-| ---------------- | ----------------- |
-| Chrome Android   | Chrome            |
-| Edge             | Internet Explorer |
-| Firefox Android  | Firefox           |
-| Opera            | Chrome            |
-| Opera Android    | Chrome Android    |
-| Safari iOS       | Safari            |
-| Samsung Internet | Chrome Android    |
-| WebView          | Chrome Android    |
+The <browser> argument is the destination browser that values will be copied to. The script automatically determines what browser to copy from based upon the destination (see table below), but manual specification is possible through the `--source=""` argument. (Mirroring data for Edge ignores this argument.)
```

Comment:
Where is the "Mirroring data for Edge ignores this argument" bit in the implementation? I can't spot it, but in any case I don't think that's the behavior we should have. Let's have `--source` override the special mirroring logic for Edge if it's used. (If it's a no-op, it would be better to have the command fail with an error message.)

Reply:
Actually, thinking about it, maybe we should remove this argument entirely?  In all honesty, the sole purpose of this argument was to allow for mirroring in Edge from either IE or Chrome -- it's an argument that's not exactly well-supported and allows for mirroring across different engines.  We wouldn't want to encourage someone trying to mirror Internet Explorer's data onto Safari, for example.

Reply:
I've never used it, so I wouldn't mind. It does seem a little bit plausible that someone might want to mirror Opera to Opera Android instead of using Chrome Android, in case a feature was added (or removed) from both. I'll defer to @ddbeck on whether this should stay or not.

Reply:
Resolved by commit d6a5b11af19cb22f0726f49c34bcbd550a60deec.